lblod/frontend-loket
# Add a conversation creation route

This will allow users to create new test conversations. The route is only accessible from the "controle" environment.

To test:
- run `npm run start:controle -- --proxy https://dev.loket.lblod.info`
- go to the berichtencentrum module
- click the new "nieuwe conversatie" button
- create a new conversation
- the edit page should now also show the conversation uri in an accordion

All of this should only be possible in the controle environment. The regular loket build should hide everything.
